mne_bids/tests/test_dig.py

```diff
@@ -325,3 +325,21 @@ def test_convert_montage():
     assert pos['coord_frame'] == 'mri'
     assert_almost_equal(pos['ch_pos']['EEG 001'],
                         [-0.0313669, 0.0540269, 0.0949191])
+
+
+def test_electrodes_io(tmp_path):
+    """Ensure only electrodes end up in *_electrodes.json."""
+    raw = _load_raw()
+    raw.pick_types(eeg=True, stim=True)  # we don't need meg channels
+    bids_root = tmp_path / 'bids1'
+    bids_path = _bids_path.copy().update(root=bids_root, datatype='eeg')
+    (bids_root / 'sub-01' / 'ses-01' / 'eeg').mkdir(parents=True,
+                                                    exist_ok=True)
```

Comment:
```suggestion
```

As written in the comment above, let's try without creating the folder manually, `write_raw_bids` should do it.

Reply:
Ok! wil commit this and see if it passes the tests. and I will add the UTF-8 encoding, thank you for reminding me about that.